src/Helper/ValidateStreet.php

```diff
@@ -36,7 +36,13 @@ class ValidateStreet
      */
     public static function validate(string $fullStreet, string $localCountry, ?string $destinationCountry): bool
     {
-        $result = preg_match(ValidateStreet::getStreetRegexByCountry($localCountry, $destinationCountry), $fullStreet, $matches);
+        $regex = ValidateStreet::getStreetRegexByCountry($localCountry, $destinationCountry);
```

Comment:
^ inderdaad

Reply:
Waarom kan $destinationCountry eigenlijk null zijn?

Reply:
Dit kan als je een zending hebt die buiten NL of BE valt, dan is er geen regex om toe te passen.

Reply:
Return dan maar voor regel 39 true als $destinationCountry null is